src/depfile_parser_test.cc

```diff
@@ -101,6 +101,18 @@ TEST_F(DepfileParserTest, Spaces) {
             parser_.ins_[2].AsString());
 }
 
+TEST_F(DepfileParserTest, BackslashesSpaces) {
+  // Backslashes are unescaped, except when they follow whitespace or a hash sign.
```

Comment:
Sorry for being super nitpicky, but could you s/whitespace/spaces, tabs,/? I didn't realize on first read that whitespace here includes tabs, hence my question above. Maybe add a tab to the test string too.

Reply:
I appreciate your feedback, nitpicks are fine!

There is also an existing `Escapes` test that already checks for `\\#`. The new [back]`SlashesSpaces` test was intended to specifically check for one or more backslashes followed by slashes/spaces. For completeness, the comment also mentions hash sign (these are the only characters that can be escaped).

Should I, instead, modify the Escapes test or maintain this new test that demonstrates that multple backslashes are not consumed (unless followed by a space)?

Would you prefer it if I rebase the series on master and squash the fixup changes in the first patch?